### PR TITLE
Fixed syntax error in prometheus-adapter

### LINKED ISSUE #7293: Incorrect label syntax in prometheus-adapter manifest
**Is this a request for help?**:

---

**BUG REPORT ** :

**Version of Helm and Kubernetes**:
Helm: 2.7.1 K8S: 1.9.3

**Which chart**:
prometheus-adapter

**What happened**:
```
Error while deploying chart: rpc error: code = Unknown desc = error validating  : error validating data: found invalid field label for v1.ObjectMeta
```

**What you expected to happen**:
Chart to be deployed

**How to reproduce it** (as minimally and precisely as possible):
Simple helm deploy

**Anything else we need to know**:
This looks like a syntax error:
https://github.com/helm/charts/blame/master/stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml#L6 (should be labels rather than label ?)

### PR DESCRIPTION
**Pull request by mateiidavid**
**What this PR does / why we need it**:
Fixes syntax error in `prometheus-adapter` chart

**Which issue this PR fixes** : fixes #7293

### CODE REVIEW

**Comment by mattjmcnaughton** [comment]
I don't have approve rights, but this looks good to me :)

**Comment by hairyhenderson** [comment]
@Matei207 there's another occurrence in https://github.com/helm/charts/blob/98b48cb722047701477029658a4a5b78962c4d7f/stable/prometheus-adapter/templates/secret.yaml#L5 that maybe could be fixed too?

File: stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   annotations:
 {{ toYaml .Values.service.annotations | indent 4 }}
-  label:
+  labels:
```
Comment (davidkarlsen) [inline stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml:6]: Please fix https://github.com/helm/charts/blob/98b48cb722047701477029658a4a5b78962c4d7f/stable/prometheus-adapter/templates/secret.yaml#L5 too - as was mentioned in the comments

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml (hunk 1 above)
Comment (davidkarlsen) [reply stable/prometheus-adapter/templates/custom-metrics-apiserver-service.yaml:6]: @Matei207 bump

**Comment by Matei207** [comment]
Hi @davidkarlsen, sorry for the long wait, I was on holiday. Thanks for pointing it out to me, I've updated the PR to also include the fix in `secret.yaml`. 

Let me know if there is anything else that's missing!

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**